### PR TITLE
Add features to datepicker

### PR DESCRIPTION
**Pull request by youda97**

### CODE REVIEW

File: src/datepicker/datepicker.component.ts [hunk 1]
```diff
@@ -141,6 +164,9 @@ export class DatePicker implements OnDestroy {
 		// add day classes and special case the "today" element based on `this.value`
 		Array.from(dayContainer).forEach(element => {
 			element.classList.add("bx--date-picker__day");
+			if (!this.value) {
```
Comment (cal-smith) [inline src/datepicker/datepicker.component.ts]: Just curious what case this is handling?

Comment (youda97) [reply src/datepicker/datepicker.component.ts]: When there is no assigned value I get a `TypeError: Cannot read property 'length' of undefined`

Comment (cal-smith) [reply src/datepicker/datepicker.component.ts]: Ah, okay ... maybe a short comment noting that?

Comment (youda97) [reply src/datepicker/datepicker.component.ts]: Well it's a simple function guard so not really sure what to add as a comment

**Pull request merged**